app/src/main/java/org/tokend/template/features/trade/orderbook/view/adapter/OrderBookEntryViewHolder.kt

```diff
@@ -19,6 +24,20 @@ class OrderBookEntryViewHolder(view: View,
 
     private var isBuy = false
 
+    fun bind(item: OrderBookEntryListItem, clickListener: SimpleItemClickListener<OrderBookEntryListItem>?, maxVolume: BigDecimal) {
+        super.bind(item, clickListener)
+
+        val maxValue = BigDecimalUtil.scaleAmount(maxVolume,
```

Comment:
Do not cut off decimal part. Perform division and multiplication by 100 in BigDecimals with MathContext.DECIMAL_128 and only transform it into Int in the end